### PR TITLE
Add intermediary 'data' folder to folder structure sans files exercise

### PR DESCRIPTION
**Pull request by JCSzamosi**
Please delete the text below before submitting your contribution. 
---
The exercise was misleading since the question did not mention the 'data' folder.

### CODE REVIEW

File: _episodes/03-create.md [hunk 1]
```diff
@@ -712,10 +712,10 @@ but it does find the copy in `thesis` that we didn't delete.
...
 >
 > Assume that the file structure is in a folder called '2016-05-18-data',
-> which contains folders named 'raw' and 'processed' that contain data files.
-> The goal is to copy the file structure of the `2016-05-18-data` folder
-> into a folder called `2016-05-20-data` and remove the data files from
-> the directory you just created.
+> which contains a 'data' folder that in turn contains folders named 'raw' and
+> 'processed' that contain data files.  The goal is to copy the file structure
+> of the `2016-05-18-data` folder into a folder called `2016-05-20-data` and
+> remove the data files from the directory you just created.
```
[the remark this answers is not in the document]
Comment (JCSzamosi) [reply _episodes/03-create.md]: Sure. I'm not sure what the process here is. Do you want me to make that change and then resubmit?

[the remark this answers is not in the document]
Comment (shwina) [reply _episodes/03-create.md]: Sorry @JCSzamosi for the late response. Just add and push commits to the branch `JCSzamosi:gh-pages`, and they will be reflected as updates to this PR. Please let me know if you need any help!

[the remark this answers is not in the document]
Comment (JCSzamosi) [reply _episodes/03-create.md]: Done. Please let me know if there are any problems.

**Comment by shwina** [comment]
Thank you!

**Pull request merged**